frappe/utils/dashboard.py

```diff
@@ -85,13 +85,8 @@ def get_dashboards_with_link(docname, doctype):
 
 
 def sync_dashboards(app=None):
-	"""Import, overwrite fixtures from `[app]/fixtures`"""
-	if not cint(frappe.db.get_single_value("System Settings", "setup_complete")):
-		return
```

Comment:
reference: https://github.com/frappe/erpnext/issues/21669

Reply:
now that i think about it we could've left it in there as well ..as this gets triggered on setup complete anyway :P

the 2 cases are:
1. someone installed an app on a fresh non setup completed site
2. installed app on a setup completed site

for case 1: dashboards would get synced on setup complete by setup wizard
for case2: dashboards would get synced when installing the app